src/com/amplitude/api/DatabaseHelper.java

```diff
@@ -47,19 +49,43 @@
             + EVENT_FIELD + " TEXT);";
 
     private File file;
+    private String instanceName;
 
     private static final AmplitudeLog logger = AmplitudeLog.getLogger();
 
+    @Deprecated
     static synchronized DatabaseHelper getDatabaseHelper(Context context) {
-        if (instance == null) {
-            instance = new DatabaseHelper(context.getApplicationContext());
+        return getDatabaseHelper(context, null);
+    }
+
+    static synchronized DatabaseHelper getDatabaseHelper(Context context, String instance) {
+        if (TextUtils.isEmpty(instance)) {
+            instance = Constants.DEFAULT_INSTANCE;
+        }
+        instance = instance.toLowerCase();
```

Comment:
Maybe these few lines of logic should be refactored into a `normalizeInstanceName` method?